pkg/migrations/migration_version.go

```diff
@@ -45,16 +53,51 @@ func Read(dbPath string) (*MigrationVersion, error) {
 		return nil, err
 	}
 
+	// Get the last time file was updated
+	fileInfo, err := os.Lstat(path)
```

Comment:
Is it possible that path is a symbolic link? Can you confirm if we can get correct mod time here.

Reply:
Turns out I'm not using this value from here.  I'm pulling it in the replica manager as I can get the path and not worry about the links.  I'm going to remove this one.